build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-THIS_DIR="$(readlink -m "$(dirname "$0")")"
+THIS_DIR="$(pwd)"
```

Comment:
The main issue with 'pwd' is that it assumes that you are launching the script from the root as ./build.sh. If you launch it from other place it won't work.

We can leave it like this for now but I'll try to find a more robust solution

Reply:
Could you try this?

THIS_DIR="$(cd "$(dirname "$0")" && pwd -P && cd - > /dev/null)"

If it works we can put it in some other places

Reply:
works 👍 . I'm updating mac script to use this

Reply:
Great! Can you update `build.sh` and `log.sh` as well?